tests/ounit/lustre/lustreSyntaxChecks/misplaced_frame_block.lus

```diff
@@ -0,0 +1,24 @@
+node test(x: int) returns(y1, y2: int);
+let
+  check (y1 = 2 or y1 = 3);
+
+if oscillate()
+then
+   frame ( y2 )
+   y2 = 3;
+   let
+      if x > 0
+      then
+         y2 = 4;
+      fi
+   tel
+   y1 = 2;
+else
+   y1 = 3;
+fi
+tel;
+
+node oscillate() returns (Z: bool)
+let
+   Z = true -> not pre Z;
+tel;
```

Comment:
Kind 2 returns the following error for this input: `Frame blocks are not allowed inside other frame blocks`

The message should say something like:
`Frame blocks are not allowed inside other frame blocks or if statements`